manifests/backup/mysqldump.pp

```diff
@@ -33,6 +33,7 @@
   $compression_command      = 'bzcat -zc',
   $compression_extension    = '.bz2',
   $backupmethod_package     = undef,
+  Array[Strings] $excludedatabases = [],
```

Comment:
We can indent the other parameters as well.

Reply:
If you could indent the other parameters, that would be great. Once that is done, I will happy to merge this PR 👍